### DISCUSSION TITLE
Command + Scroll Zooming not working on MX Master

### DISCUSSION THREAD

**Discussion by IrfanNian** [noah-nuebling/mac-mouse-fix#266] [Questions] [answered]
Hi,

I am using MX Master 2s. I cannot get the Command + Scroll to work even after adding apps in the App-Specific settings. All the other button (eg. Middle Button) shortcut works.

**Comment by Metagit** [comment]
Hey Irfan, I'm having the same issue. I just installed Mac-mouse-fix only for this single feature, since BTT couldn't make it work too.
Did you find a solution yet? I really need this feature at all cost, but there seems to be no way whatsoever. Very frustrating.

**Comment by chansearrington** [comment]
I'm actually here with the exact same problem.

I've got an MX Master 3

I'm trying to use an iPad app (ArcSite) on my M1 Mac.

I'm able to zoom in/out on the app with my track pad, but unable to do so with my mouse.

I downloaded mac mouse fix hoping it would fix this issue based on a forum post I saw on Stack.

It does not, and I've set the app specific settings as well like @Metagit and @IrfanNian

**Comment by SeBr28** [comment]
Same here. MX Master 3. Mac OS Monterey 12.5 - Mac mini M1
I tried different things and yes also deleten preview and adding it again. Fully stopping preview and so on..
No matter how hard I scroll.. nothing zooms :( So +1 here in problem stated above

**Comment by noah-nuebling** [comment]
Hey guys, thanks for reaching out! 

I'm not sure what might be causing these issues, as I can't reproduce them, but one thing I noticed is that you all seem to be using Logitech Mice. So maybe you still have Logitech Options or some other driver installed that interferes with things in the background? Or maybe your mice are sending non-standard scroll events that Mac Mouse Fix can't pick up on? If that's the case, you might be able to install Logitech Options, then turn off all the fancy scroll settings, and then uninstall Logitech Options again. This might make your mouse send normal scroll events that Mac Mouse Fix can process. If you get the chance you could also test with a non-Logitech mouse and see if that works. Or you could also open a [Bug Report](https://noah-nuebling.github.io/mac-mouse-fix-feedback-assistant/?type=bug-report) and attach some Console Logs. 

Any of these things would be very helpful in figuring out what's going on. So do let me know if you find our more!

**Comment by noah-nuebling** [comment, marked as answer]
Hey! Just marking this as answered to clean things up since @chansearrington has created Issue #314 from this discussion.

Please refer to #314 for future discussion of the problem. Thanks!

**Comment by s-smits** [comment]
This problem is still occuring.

**Comment by AntonTal** [comment]
Same problem with MX Master 4
